amgut/db/scrub.sql

```diff
@@ -0,0 +1,343 @@
+-- This script is used to scrub the live American Gut database in order to
+-- generate a test database for Travis without personal information but
+-- representative of the real world.
+
+-- Function to generate a string with all the different characters present in
+-- a column (adapted from: http://dba.stackexchange.com/a/83679)
+CREATE FUNCTION retrieve_chars(column_name varchar, table_name varchar) RETURNS varchar AS $$
```

Comment:
Oh, I see, this is over all rows. [a-z] probably not necessary, but asserting independently that this method handles non-utf8 as expected is still a good idea.